runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -169,21 +193,58 @@ private void init()
+		add(combo(5, heal(PRAYER, perc(0.1, 2)), boost(MAGIC, perc(0.05, 2)), heal(ATTACK, perc(0.1, -2)), heal(STRENGTH, perc(0.1, -2)), heal(DEFENCE, perc(0.1, -2))),
+			ANCIENT_BREW4, ANCIENT_BREW3, ANCIENT_BREW2, ANCIENT_BREW1);
+		add(combo(5, boost(ATTACK, perc(0.15, 5)), boost(STRENGTH, perc(0.15, 5)), boost(DEFENCE, perc(0.15, 5)), boost(RANGED, perc(0.1, 4)), boost(MAGIC, perc(0.1, 1))),
+			CASTLEWARS_BREW4, CASTLEWARS_BREW3, CASTLEWARS_BREW2, CASTLEWARS_BREW1);
+		add(combo(2, boost(ATTACK, perc(0.15, 5)), boost(STRENGTH, perc(0.15, 5))),
+			SUPER_COMBAT_POTION4_23543, SUPER_COMBAT_POTION3_23545, SUPER_COMBAT_POTION2_23547, SUPER_COMBAT_POTION1_23549 /* LMS */);
+
+		// Barbarian Herblore
+		add(food(3), ANTIPOISON_MIX1, ANTIPOISON_MIX2, RELICYMS_MIX1, RELICYMS_MIX2);
+		add(food(6), ANTIPOISON_SUPERMIX1, ANTIPOISON_SUPERMIX2, ANTIDOTE_MIX1, ANTIDOTE_MIX2, ANTIFIRE_MIX1, ANTIFIRE_MIX2,
+			EXTENDED_ANTIFIRE_MIX1, EXTENDED_ANTIFIRE_MIX2, SUPER_ANTIFIRE_MIX1, SUPER_ANTIFIRE_MIX2, EXTENDED_SUPER_ANTIFIRE_MIX1, EXTENDED_SUPER_ANTIFIRE_MIX2);
+		add(combo(food(3), boost(ATTACK, perc(.10, 3))), ATTACK_MIX1, ATTACK_MIX2);
+		add(combo(food(3), boost(STRENGTH, perc(.10, 3))), STRENGTH_MIX1, STRENGTH_MIX2);
+		add(combo(5, food(3), heal(ATTACK, perc(.30, 10)), heal(STRENGTH, perc(.30, 10)), heal(DEFENCE, perc(.30, 10)), heal(RANGED, perc(.30, 10)), heal(MAGIC, perc(.30, 10))), RESTORE_MIX1, RESTORE_MIX2);
+		add(combo(food(6), boost(RUN_ENERGY, 10)), ENERGY_MIX1, ENERGY_MIX2);
+		add(combo(food(6), boost(DEFENCE, perc(.10, 3))), DEFENCE_MIX1, DEFENCE_MIX2);
+		add(combo(food(6), boost(AGILITY, 3)), AGILITY_MIX1, AGILITY_MIX2);
+		add(combo(2, food(6), boost(ATTACK, perc(.10, 3)), boost(STRENGTH, perc(.10, 3))), COMBAT_MIX1, COMBAT_MIX2);
+		add(new PrayerMix(), PRAYER_MIX1, PRAYER_MIX2);
+		add(combo(food(6), boost(ATTACK, perc(.15, 5))), SUPERATTACK_MIX1, SUPERATTACK_MIX2);
+		add(combo(food(6), boost(FISHING, 3)), FISHING_MIX1, FISHING_MIX2);
+		add(combo(food(6), boost(RUN_ENERGY, 20)), SUPER_ENERGY_MIX1, SUPER_ENERGY_MIX2);
+		add(combo(food(6), boost(HUNTER, 3)), HUNTING_MIX1, HUNTING_MIX2);
+		add(combo(food(6), boost(STRENGTH, perc(.15, 5))), SUPER_STR_MIX1, SUPER_STR_MIX2);
+		add(combo(food(6), boost(MAGIC, 3)), MAGIC_ESSENCE_MIX1, MAGIC_ESSENCE_MIX2);
+		add(new SuperRestoreMix(), SUPER_RESTORE_MIX1, SUPER_RESTORE_MIX2);
+		add(combo(food(6), boost(DEFENCE, perc(.15, 5))), SUPER_DEF_MIX1, SUPER_DEF_MIX2);
+		add(combo(food(6), boost(RANGED, perc(.10, 4))), RANGING_MIX1, RANGING_MIX2);
+		add(combo(food(6), boost(MAGIC, 4)), MAGIC_MIX1, MAGIC_MIX2);
+		add(combo(3, boost(ATTACK, perc(.20, 2)), boost(STRENGTH, perc(.12, 2)), heal(PRAYER, perc(.10, 0)), new BoostedStatBoost(DEFENCE, false, perc(.10, -2)), new BoostedStatBoost(HITPOINTS, false, perc(-.12, 6))), ZAMORAK_MIX1, ZAMORAK_MIX2);
```

Comment:
I had mentioned this on Discord, this is not the correct way to handle Zamorak Mix boosts--the hitpoints boost is incorrectly calculated here because the "heal" of the mixed potion is calculated separately from the drain of the zamorak brew, both against the starting boosted hp value. My PR #4036 handles this and has a test case showing the expected behavior. I'm inclined to prefer that implementation for barbarian potions to duplicating all of the potion effects for each entry and creating prayer/super restore/stamina mix classes specifically for those potions.